docs/reference/commandline/build.md

```diff
@@ -373,6 +374,21 @@ the command line.
 > repeatable builds on remote Docker hosts. This is also the reason why
 > `ADD ../file` will not work.
 
+### Build with Secrets
+
```

Comment:
I think there should be some comparison between the build secret and the swarm secret, but it can be another PR